.gitignore

```diff
@@ -10,3 +16,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+>>>>>>> 93db8985a79e7131f36d0fffb1ace31e8c14bc85
```

Comment:
The gitgnore file seems to be screwed up could you please clean it and merge again